docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -0,0 +1,194 @@
+
+In this version, Secure Credential Store (SCS) Plug-in is deprecated. The `zowe scs` and `zowe config` command groups are obsolete. The equivalent functionality that encrypts your credentials is now included in the core CLI.
+
+With the new configuration, the CLI prompts you to enter username and password securely by default. Commands in the new `zowe cnfg` command group let you manage security for any option value.
+
+## Installing @next version
+
+To get started, install the Zowe CLI `@next` version from the online registry. You can follow this procedure for a first-time installation, or to update a currently installed version.
+
+**Follow these steps:**
+
+1. Meet the [software requirements for Zowe CLI](https://docs.zowe.org/stable/user-guide/systemrequirements.html#zowe-cli-requirements).
+
+2. To install or update the core CLI, open a command-line window and issue the following command:
+
+   ```
+   npm install -g @zowe/cli@next
+   ```
+
+3. Meet the [software requirements for each plug-in](https://docs.zowe.org/stable/user-guide/cli-swreqplugins.html#software-requirements-for-zowe-cli-plug-ins).
+
+4. To install or update the Zowe CLI plug-ins, issue the following command:
+
+    ```
+    zowe plugins install @zowe/cics@next @zowe/zos-ftp-for-zowe-cli@next @zowe/ims@next @zowe/mq@next @zowe/db2@next
+    ```
+
+   The `@next` version of Zowe CLI and plug-ins are installed!
+
+5. If you previously had an instance of Zowe CLI installed, your old configuration files are no longer used in this version. Delete the following files from your local `.zowe/` directory:
+   - `.zowe/settings/imperative.json`
+   - `.zowe/profiles`
+
+   **Important!** Prior to deleting the contents of the `/profiles` directory, take note of any mainframe service details that you need (host, port, etc...). You might want to save the entire `/profiles` directory to another location on your computer so that you can reference or restore the profiles later.
+
+6. If you previously had the Secure Credential Store plug-in installed, uninstall it now to avoid unexpected behavior. Issue the following command:
+
+    ```
+    zowe plugins uninstall @zowe/secure-credential-store-for-zowe-cli
+    ```
+
+You can now configure the CLI and issue commands.
+## Initializing global configuration
+
+To begin, define a connection to z/OSMF and initialize your configuration files. We recommend this method for getting started, but you can choose create the configuration files manually if desired.
+
+**Follow these steps:**
+
+1. Issue the following command:
+
+   ```
+   zowe cnfg init --global
+   ```
+
+   The CLI provides a series of prompts.
+
+2. Respond to the prompts to enter a service name, username, and password for a mainframe service such as z/OSMF. The `--global` option ensures that your credentials are stored securely on your computer by default.
+
+   After you respond to the prompts, the following two files are added to your local `.zowe/settings` directory:
```

Comment:
Only `zowe.config.json` is added via `zowe config init --global` and it is added to `.zowe`. To generate `zowe.config.user.json` in that global location, you would need to issue `zowe config init --global --user`. Also, it isn't an exact copy. The top level `defaults`, `plugins`, and `secure` fields are empty. And the profiles do not have any properties. After generating the user config, users can add their preferences as properties in the profiles to override properties in `zowe.config.json` or add new ones.

Reply:
Ohhh I see. Thanks for your feedback on this so far, I'll have time to fix things up while folks are on holiday.  

Also need to further a couple sections towards the end (sharing global config I am especially unsure of), need to find a demo recording or get some info from team about that.

Reply:
In order to share global config, I would send you a `zowe.config.json` file and you would manually place it in your `~/.zowe` folder. 

In order to share project config, I would commit the project's `zowe.config.json` to source control and then you would pull the project to your local machine. Any command you issued from within that directly would use the project's `zowe.config.json` and the global config would simply be ignored. Very useful for test automation or CI/CD projects.